app/billing_client_test.go

```diff
@@ -0,0 +1,265 @@
+package app
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"testing"
+
+	pb "go.viam.com/api/app/v1"
+	"go.viam.com/test"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"go.viam.com/rdk/testutils/inject"
+)
+
+const (
+	subtotal                     = 37
+	sourceType                   = SourceTypeOrg
+	usageCostType                = UsageCostTypeCloudStorage
+	cost                 float64 = 20
+	discount             float64 = 9
+	totalWithDiscount            = cost - discount
+	totalWithoutDiscount float64 = cost
+	email                        = "email"
```

Comment:
This is always where I use funny strings like `oscarthegrouch@sesamestreet.xyz` but I appreciate the professionalism 🙏🏻 😆 .

Reply:
You made me burst out laughing omg. I love it